app/views/statistics/index.html.erb

```diff
@@ -10,6 +10,12 @@
   </div>
 </header>
 
+
+<%= render "govuk_publishing_components/components/notice", {
+    title: "This page is going away soon.",
+    description_govspeak: "<p class=\"govuk-body\">It's being replaced by <a href=\"https://www.gov.uk/search/statistics/\">research and statistics search.</a></p>"
```

Comment:
I think this should be `/search/research-and-statistics`, otherwise we'll send people to a redirect.

Reply:
fixed